### PR TITLE
Updated README + RDS Pass Updated Rule

### PR DESCRIPTION
**Pull request by jacknagz**
## Changes

* New Rule: AWS RDS Master Pass Changed
* Updated README with latest marketing language + fixed small errors + removed old directory listing

## Testing

* Developed and live-tested in Console
* Validated via local rule testing

### CODE REVIEW

**Review by calkim-panther — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -40,43 +38,31 @@ Each folder contains detections in the format of `<log/resource type>_<detecton_
...
-
-[Other SaaS Services](indexes/saas.md)
-
-[Panther Standard Detections](indexes/standard.md)
-
 ## Configure your Python environment
+
 ```bash
 make install
 make venv
```
Comment (edyesed) [inline README.md:45]: actually don't need `make venv` anymore. make install does what is needed

File: README.md [hunk 2]
```diff
@@ -40,43 +38,31 @@ Each folder contains detections in the format of `<log/resource type>_<detecton_
...
 pipenv shell # Optional, this will spawn a subshell containing pipenv environment variables. Running pipenv run before commands becomes optional after this step
 ````
+
 ### Install dependencies and run your first test!
+
 ```bash 
 make deps
```
Comment (edyesed) [inline README.md:52]: `make deps` is now called `make install`

File: rules/aws_cloudtrail_rules/aws_rds_master_pass_updated.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+from panther_base_helpers import deep_get
+
+
+def rule(event):
+    master_user_pass = deep_get(
```
Comment (edyesed) [inline rules/aws_cloudtrail_rules/aws_rds_master_pass_updated.py:5]: I suggest filtering before deep_get. There's no chance that the deep_get will return results without the eventname and source matching.

File: rules/aws_cloudtrail_rules/aws_rds_master_pass_updated.yml [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+    - p_any_aws_arns
+    - p_any_aws_account_ids
+LogTypes:
+    - AWS.CloudTrail
+RuleID: AWS.RDS.MasterPasswordUpdated
+Threshold: 1
+Tests:
+    - ExpectedResult: false
+      Log:
+        awsRegion: us-west-1
```
Comment (edyesed) [inline rules/aws_cloudtrail_rules/aws_rds_master_pass_updated.yml:24]: I am unsure of the implications of not wrapping the log event in `{}` . We received another PR with this non-curly'ed format, and we do now know of possible downstream impact from missing the curlies. 

Additionally, the test's Name attribute is historically at top-of-test, which helps to identify when manually troubleshooting.

**Review by edyesed — COMMENTED** [review]
comments. The make targets referenced in the README still apply, so there is no harm in continuing to reference them at this point in time.

File: rules/aws_cloudtrail_rules/aws_rds_master_pass_updated.yml (hunk 4 above)
Comment (jacknagz) [reply rules/aws_cloudtrail_rules/aws_rds_master_pass_updated.yml:24]: There's no implication of using YML IMO, it's the same thing. FWIW, this is the result when you download from the UI.

File: rules/aws_cloudtrail_rules/aws_rds_master_pass_updated.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
+from panther_base_helpers import deep_get
+
+
+def rule(event):
+    return (
```
Comment (jacknagz) [inline rules/aws_cloudtrail_rules/aws_rds_master_pass_updated.py:5]: @edyesed I'm opting for readability and simplicity.

**Review by calkim-panther — APPROVED** [review]

File: rules/aws_cloudtrail_rules/aws_rds_master_pass_updated.py (hunk 3 above)
Comment (calkim-panther) [reply rules/aws_cloudtrail_rules/aws_rds_master_pass_updated.py]: I believe python automatically will exit execution if the first statement evaluates to be false.

**Review by edyesed — APPROVED** [review]
This change set updates the README to point to most current generation docs domain and make commands. 

This also updates the templates/example* files with new content and extra commentary guidance 💪 

Additionally, this adds a cloudtrail based detection for RDS password changes.

**Pull request merged**